package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "revai-node-sdk",
-  "version": "2.0.1",
+  "version": "2.1.0",
```

Comment:
I did a minor version update since this changes the functionality of the SDK a reasonable amount but doesn't break existing implementations